fsm/interceptors.go

```diff
@@ -150,7 +150,8 @@ func ManagedContinuationsWithJitter(historySize int, maxSizeJitter int, workflow
 				}
 			}
 
-			historySizeExceeded := int64(historySize+rng.Intn(maxSizeJitter)) < *decision.Events[0].EventId
+			eventCount := *decision.Events[0].EventId
```

Comment:
Are we confident that AWS will never change how they implement this?  In their docs it just says `EventId` is unique.  I don't see a reason for them to change, but perhaps safer to just look at the length of `Events`?